src/__tests__/HospitalRun.test.tsx

```diff
@@ -39,30 +30,29 @@ describe('HospitalRun', () => {
       breadcrumbs: { breadcrumbs: [] },
       components: { sidebarCollapsed: false },
     } as any)
-    const wrapper = mount(
+
+    const Wrapper = ({ children }: { children: ReactElement }) => (
```

Comment:
unless the types are blocking you from converting, let's focus on converting.

Reply:
I do get an implicit any if I do not do it that way, I can use React.FC, though it is not recommended.